paddle/fluid/framework/ir/conv_bn_fuse_pass.cc

```diff
@@ -44,6 +44,20 @@ namespace ir {
   GET_IR_NODE_FROM_SUBGRAPH(bn_saved_mean, bn_saved_mean, pattern_name);     \
   GET_IR_NODE_FROM_SUBGRAPH(bn_saved_variance, bn_saved_variance, pattern_name)
 
+// Check environment flag whether MKL-DNN should be used
+// TODO(wojtuss): remove after the "OP placement" pass is added
+static bool IsMKLDNNSetOn() {
+  const char* flag = std::getenv("FLAGS_use_mkldnn");
+  if (flag) {
+    std::string flag_str(flag);
+    std::transform(flag_str.begin(), flag_str.end(), flag_str.begin(),
+                   ::toupper);
+    return !flag_str.compare("ON") || !flag_str.compare("TRUE") ||
+           !flag_str.compare("1");
+  }
+  return false;
+}
+
```

Comment:
Is it possible to move this function to a common file? I think it could be useful is some other places as well (such as CAPI applications).

Reply:
It won't be necessary. This function will be removed soon as the MKL-DNN-placement pass (to appear) will provide information on whether MKL-DNN should be used or not.